conn.go

```diff
@@ -118,10 +130,12 @@ func (c *Conn) User() User {
 	return c.user
 }
 
+// Setting the user resets any message beng generated
 func (c *Conn) SetUser(user User) {
 	c.locker.Lock()
 	defer c.locker.Unlock()
 	c.user = user
+	c.msg = &message{}
```

Comment:
I didn't see any case where we wanted to set the user and *not* set the message to empty. Since I was introducing a new site where the user is set, I figured I should make the dependency explicit.

Reply:
Aye, that makes sense.